ui/src/components/DownloadListItem/index.js

```diff
@@ -2,31 +2,48 @@ import React from 'react';
 import { withPolling } from 'ui/utils/hocs';
 import { compose, withProps } from 'recompose';
 import OptionListItem from 'ui/components/OptionListItem';
+import { activeOrganisationSettingsSelector } from 'ui/redux/modules/auth';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import createdAt from 'ui/containers/Owner/CreatedAt';
 
-const DownloadListItem = ({ model, deleteModel }) => {
-  if (model.get('isReady')) {
+const DownloadListItem = (props) => {
+  const { download, deleteDownloadModel, exporationExport } = props;
+
+  const expireExports = moment(download.get('time')).add(exporationExport.get('ttl'), 'seconds');
+  const formattedExpireExports = createdAt(expireExports, 'Expires');
+
+  if (download.get('isReady')) {
     return (
       <OptionListItem
-        label={model.get('name')}
-        href={model.get('url')}
+        label={`${download.get('name')} - (${formattedExpireExports})`}
```

Comment:
We'll need to handle here if there is no expiry date for this download - aka `expiresAt` is set to null

Reply:
(See below comment)